subcommands/devices/config_updates.go

```diff
@@ -32,7 +32,9 @@ currently configured and reporting.`,
 	configCmd.AddCommand(configUpdatesCmd)
 	configUpdatesCmd.Flags().StringP("tag", "", "", "Target tag for device to follow")
 	configUpdatesCmd.Flags().StringP("tags", "", "", "Target tag for device to follow")
+	configUpdatesCmd.Flags().BoolP("inherit-tag", "", false, "Toggle if tag should be inherited from parent")
 	configUpdatesCmd.Flags().StringP("apps", "", "", "comma,separate,list")
+	configUpdatesCmd.Flags().BoolP("inherit-apps", "", false, "Toggle if apps should be inherited from parent")
```

Comment:
I'm not sure parent is the correct word for this. Also - should `apps` and `inherit-apps` be mutually exclusive? Same thing with tags as well.